webapp/src/components/sidebar/playbooks_sidebar.tsx

```diff
@@ -0,0 +1,133 @@
+    direction: 'desc',
+    statuses: [PlaybookRunStatus.InProgress, PlaybookRunStatus.Finished],
+};
+
+interface PlaybookSidebarProps{
+    team_id: string;
+}
+
+const PlaybooksSidebar = (props: PlaybookSidebarProps) => {
+    const groups: Array<SidebarGroup> = [];
+
+    const [playbooks] = usePlaybooksCrud({team_id: props.team_id, per_page: 20});
+    const [playbookRuns] = useRunsList(defaultRunsFetchParams);
+
+    // Not a correct list of playbooks, should be changed
+    const playbooksItems = playbooks ? playbooks.map((playbook) => {
+        return {
+            areaLabel: playbook.title,
+            className: '',
+            display_name: playbook.title,
+            icon: playbook.public ? <StyledPlaybookIcon/> : <StyledPrivatePlaybookIcon/>,
+            isCollapsed: false,
+            itemMenu: null,
+            link: `/playbooks/playbooks/${playbook.id}`,
+        };
+    }) : [];
+
+    const playbooksGroup: SidebarGroup = {
+        collapsed: false,
+        display_name: 'Playbooks',
+        id: 'playbooks',
+        items: playbooksItems,
+    };
+
+    // Not a correct list of runs, should be changed
+    const runsItems = playbookRuns ? playbookRuns.map((run): GroupItem => {
+        return {
+            areaLabel: run.name,
+            className: '',
+            display_name: run.name,
+            icon: <StyledPlaybookRunIcon/>,
+            isCollapsed: false,
+            itemMenu: null,
+            link: pluginUrl(`/runs/${run.id}`),
+        };
+    }) : [];
+
+    const runsGroup: SidebarGroup = {
+        collapsed: false,
+        display_name: 'Runs',
+        id: 'runs',
+        items: runsItems,
+    };
+
+    // favorite category rendered statically for the UI, should be changed
+    const favItems: Array<GroupItem> = [
+        {
+            areaLabel: 'Cool playbook',
+            className: '',
+            display_name: 'Cool playbook',
+            icon: <StyledPlaybookIcon/>,
+            isCollapsed: false,
+            itemMenu: null,
+            link: 'some',
+        },
+        {
+            areaLabel: 'Cool run',
+            className: '',
+            display_name: 'Cool run',
+            icon: <StyledPlaybookRunIcon/>,
+            isCollapsed: false,
+            itemMenu: null,
+            link: 'some',
+        },
+    ];
+
+    const fav: SidebarGroup = {
+        collapsed: false,
+        display_name: 'Favorites',
+        id: 'favorites',
+        items: favItems,
+    };
+    groups.push(fav);
+    groups.push(runsGroup);
+    groups.push(playbooksGroup);
+
+    return (
+        <Sidebar
+            groups={groups}
+            headerDropdown={null}
+            onGroupClick={() => {/*empty*/}}
+            team_id={props.team_id}
+        />
+    );
+};
+
+export default PlaybooksSidebar;
+
+const sharedIconStyles = css`
+    width: 18px;
+    height: 18px;
+`;
+
+const StyledPlaybookIcon = styled(PlaybookIcon)`
+    ${sharedIconStyles}
+`;
```

Comment:
For future pending icons, using the MDI version directly might be simpler

```suggestion
const StyledPlaybookIcon = () => (
    <Icon
        path={mdiBookOutline}
        size={1.25}
    />
);
```